<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alembicExport</class>
 <widget class="QWidget" name="alembicExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>172</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Alembic Export</string>
  </property>
  <property name="windowIcon">
   <iconset resource="maya.qrc">
    <normaloff>:/shelf/img/shelf/shelf_alembicExport35.png</normaloff>:/shelf/img/shelf/shelf_alembicExport35.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>66</y>
     <width>41</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/shelf/img/shelf/shelf_alembicExport35.png</normaloff>:/shelf/img/shelf/shelf_alembicExport35.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/denialBtn24.png</normaloff>:/btn/img/btn/denialBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>145</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/helpBtn24.png</normaloff>:/btn/img/btn/helpBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMsg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>316</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ALEMBIC EXPORT | Selectiv</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>145</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/user/img/user/_default.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblReport">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>84</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/btn/img/btn/btnCamera48.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblScript">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>54</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/shelf/img/shelf/shelf_default35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReport">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>145</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/mailBtn24.png</normaloff>:/btn/img/btn/mailBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxCam">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>85</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CAM</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssets">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>55</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ASSETS</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsChar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>85</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>char</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsProp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>115</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>prop</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsSet">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>85</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>set</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAssetsFx">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>115</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>fx</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblReport_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>114</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/shelf/img/shelf/shelf_suggestion35.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxLight">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>115</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LIGHT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxMerge">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>15</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>merge</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAll">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>ALL</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenFolder">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>144</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/folderBtn24.png</normaloff>:/btn/img/btn/folderBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportFx">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>66</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/shelf/img/shelf/shelf_alembicExportFx35.png</normaloff>:/shelf/img/shelf/shelf_alembicExportFx35.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxSelected">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>20</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>SELECTED</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtFrameEnd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>40</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1210</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtFrameStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1001</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>41</y>
     <width>8</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnExport</zorder>
  <zorder>btnCancel</zorder>
  <zorder>btnHelp</zorder>
  <zorder>edtMsg</zorder>
  <zorder>lblUser</zorder>
  <zorder>lblReport</zorder>
  <zorder>lblScript</zorder>
  <zorder>btnReport</zorder>
  <zorder>cbxCam</zorder>
  <zorder>cbxAssets</zorder>
  <zorder>cbxAssetsChar</zorder>
  <zorder>cbxAssetsProp</zorder>
  <zorder>cbxAssetsSet</zorder>
  <zorder>cbxAssetsFx</zorder>
  <zorder>lblReport_2</zorder>
  <zorder>cbxLight</zorder>
  <zorder>cbxMerge</zorder>
  <zorder>cbxAll</zorder>
  <zorder>btnOpenFolder</zorder>
  <zorder>btnExportFx</zorder>
  <zorder>cbxSelected</zorder>
  <zorder>edtFrameEnd</zorder>
  <zorder>edtFrameStart</zorder>
 </widget>
 <resources>
  <include location="maya.qrc"/>
 </resources>
 <connections/>
</ui>
